### PR TITLE
Updated downstream version numbers from 4.5 to 4.6.

### PR DESCRIPTION
**Pull request by adahms**

### CODE REVIEW

File: doc-Appliance_Hardening_Guide/miq/build/en-US/master.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE book [
 <!ENTITY % sgml.features "IGNORE">
 <!ENTITY % xml.features "INCLUDE">
-<!ENTITY % DOCBOOK_ENTS PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/dbcentx.mod">
+<!ENTITY % DOCBOOK_ENTS PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.6//EN" "http://www.oasis-open.org/docbook/xml/4.6/dbcentx.mod">
```
Comment (dayleparker) [inline doc-Appliance_Hardening_Guide/miq/build/en-US/master.xml]: Hi @adahms - looks like the find and replace picked up the Docbook version as well -- should this remain as 4.5?

File: doc-Managing_Providers/topics/Viewing_Network_Providers_Timeline.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 = Viewing a Network Provider's Timeline
 
-//SME feedback - leave section out of 4.5 release. See https://bugzilla.redhat.com/show_bug.cgi?id=1382585#c7
+//SME feedback - leave section out of 4.6 release. See https://bugzilla.redhat.com/show_bug.cgi?id=1382585#c7
```
Comment (dayleparker) [inline doc-Managing_Providers/topics/Viewing_Network_Providers_Timeline.adoc]: This perhaps should be changed back to 4.5, but am unsure about this note.

File: doc-Managing_Providers/topics/Viewing_Swift_Manager_Timeline.adoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [[viewing_swift_timeline]]
 = Viewing a Swift Manager’s Timeline
 
-//SME feedback - leave section out of 4.5 release. See https://bugzilla.redhat.com/show_bug.cgi?id=1382585#c7
+//SME feedback - leave section out of 4.6 release. See https://bugzilla.redhat.com/show_bug.cgi?id=1382585#c7
```
Comment (dayleparker) [inline doc-Managing_Providers/topics/Viewing_Swift_Manager_Timeline.adoc]: This perhaps should be changed back to 4.5, but am unsure about this note.

File: doc-Provisioning_Virtual_Machines_and_Hosts/topics/Provisioning_Requests.adoc [hunk 4]
```diff
@@ -480,7 +480,7 @@ include::common/provisioning-requests-openstack.adoc[]
 [[provisioning-a-google-compute-engine-instance-from-an-image]]
 ==== Provisioning a Google Compute Engine Instance from an Image
 
-From Red Hat {product-title} 4.5, a Google Compute Engine instance can be provisioned. Some functionality available in other cloud providers is currently unavailable when provisioning from Google Compute Engine.
+From Red Hat {product-title} 4.6, a Google Compute Engine instance can be provisioned. Some functionality available in other cloud providers is currently unavailable when provisioning from Google Compute Engine.
```
Comment (dayleparker) [inline doc-Provisioning_Virtual_Machines_and_Hosts/topics/Provisioning_Requests.adoc]: This should be reverted to 4.5 (or possibly 4.2), I believe.

File: methods_available_for_automation/Methods_Available_for_Automation.adoc [hunk 5]
```diff
@@ -215,17 +215,17 @@ The result of dumpRoot is below. The value of miq_server is what gets
...
-[----] I, [2012-10-23T13:53:54.529961 #5320:ef8c538]  INFO -- : <User-Defined Method> [InspectMe] - Root:<$evm.root> Attributes - object_name: Request
-[----] I, [2012-10-23T13:53:54.531067 #5320:ef8c538]  INFO -- : <User-Defined Method> [InspectMe] - Root:<$evm.root> Attributes - request: inspectme
-[----] I, [2012-10-23T13:53:54.534054 #5320:ef8c538]  INFO -- : <User-Defined Method> [InspectMe] - Root:<$evm.root> Attributes - vm: DEV-JaneM
-[----] I, [2012-10-23T13:53:54.535156 #5320:ef8c538]  INFO -- : <User-Defined Method> [InspectMe] - Root:<$evm.root> Attributes - vm_id: 85
-[----] I, [2012-10-23T13:53:54.536238 #5320:ef8c538]  INFO -- : <User-Defined Method> [InspectMe] - Root:<$evm.root> Attributes - vmdb_object_type: vm
-[----] I, [2012-10-23T13:53:54.537159 #5320:f329024]  INFO -- : <User-Defined Method> [InspectMe] - Root:<$evm.root> End Attributes
-[----] I, [2012-10-23T13:53:54.537772 #5320:f329024]  INFO -- : <User-Defined Method>
+[----] I, [2012-10-23T13:53:54.617279 #5320:f329024]  INFO -- : <User-Defined Method> [InspectMe] - EVM Automate Method Started
```
Comment (dayleparker) [inline methods_available_for_automation/Methods_Available_for_Automation.adoc]: @adahms Should these sixes be changed back to fives?

**Comment by dayleparker** [comment]
Hi @adahms,
There are a few things picked up by the find & replace which might need a closer look at/editing. I made an edit to a note I put in myself in the HA guide, but I think it's better if you review these other items before we merge. Everything else looks good though!
Cheers,
Dayle

**Comment by dayleparker** [comment]
Hi @adahms,
There are a few things picked up by the find & replace which might need a closer look at/editing. I made an edit to a note I put in myself in the HA guide, but I think it's better if you review these other items before we merge. Everything else looks good though!
Cheers,
Dayle

**Comment by adahms** [comment]
@dayleparker - Thanks for the review!

I have now addressed your comments by reverting some references and changing others to '4.2' instead of '4.6'.

Let me know if you spot anything else!

**Comment by dayleparker** [comment]
Thanks for making those last edits, Andrew! I gave the PR one final review and everything looks great :)
Cheers,
Dayle

**Pull request merged**